### ISSUE TITLE
Manage default plugin versions in Settings

### ISSUE THREAD

**Issue by blupu** [uncatcrea/wp-appkit#284]
Sometimes we need to override the default version for a default plugin (eg. statusbar). Lately, I ran into a problem with the statusbar plugin and I had to downgrade from 2.2.1 to 2.2.0. Right now, I have to modify the _config.xml_ file manually (see https://issues.apache.org/jira/browse/CB-12507?filter=-2).

- We could have the list of default plugins with their current version (_latest_ when not specified) in the _Settings_ panel.
- For each plugin a text input allows to specify the version to use (eg. 2.2.0)
- Letting the text input empty will default to latest version
- The specified version will translated as a _spec_ attribute in the _config.xml_ file

**Comment by lpointet** [comment]
I think for the moment, we allow to change these versions with code only (`wpak_default_phonegap_build_plugins` filter), probably because of the fact that this is a "technical" detail.

**Comment by blupu** [comment]
Interesting, do you have an example of using it?

**Comment by lpointet** [comment]
Sure. An example to fix the statusbar plugin version to 2.2.0 as in your comment above:
```
function fix_statusbar_version( $default_plugins ) {
    if( !empty( $default_plugins['cordova-plugin-statusbar'] ) ) {
        $default_plugins['cordova-plugin-statusbar']['spec'] = '2.2.0';
    }

    return $default_plugins;
}
add_filter( 'wpak_default_phonegap_build_plugins', 'fix_statusbar_version' );
```

**Comment by blupu** [comment]
You'd do a plugin for that?

**Comment by mleroi** [comment]
Yes for now you would need to put that "wpak_default_phonegap_build_plugins" filter in a plugin.
In next version (0.7) you will be able to include that directly in theme's php folder.

Note to force the version of a plugin directly from back office you can add the plugin markup in the "Plugins" field of your app, as plugins set here have priority over default ones: 
for example if you add:
```<plugin name="cordova-plugin-statusbar" spec="2.2.0" source="npm" />```
